samcli/commands/sync/command.py

```diff
@@ -496,6 +508,8 @@ def execute_watch(
     skip_infra_syncs: bool
         Boolean flag to determine if only ececute code syncs.
     """
+    # Note: skip_infra_sync is different from skip_infra_syncs, skip_infra_syncs completely skips infra syncs and
```

Comment:
Let's use another variable name that completely separates the two things

Reply:
Agreed, will change the variable that only executes code syncs to `disable-infra-syncs` for clarity.